docs/src/content/sending-messages.mdx

```diff
@@ -34,9 +34,56 @@ gets sent out. You might use personalisation to:
 - Address a user by their name
 - Provide users with a specific link to click
 - Send a unique transaction number as a follow up
+- Give users a dynamically generated list of actions they need to take
 
 Your API call must include any fields that have been set up as personalisations.
 
+### List fields
+
+A list field allows you to include a nicely formatted list of things in your
+message. A list field looks just like an ordinary field in your template.
+
+For example, your Notify template might look like this:
+
+```text
+Dear ((name)), please bring these documents to your appointment:
+
+((documents))
+```
+
+In this example, `((documents))` looks just like `((name))` but its intent is to
+become a nicely formatted list of documents.
+
+Notify allows you to convert this field into a list in the generated message.
+To do this, your code should pass in a list of things as the value of the
+personalisation field, instead of a string.
+
+For example, in Go you would pass a slice of strings:
+
+```go
```

Comment:
Is JSON possibly a language neutral/better example? i.e.

```json
{
   "name": "Kim",
   "documents": ["Driver license", "Medicare card"]
}
```

Reply:
You're right, that's better.